wallet/subnet/primary/wallet.go

```diff
@@ -120,9 +119,12 @@ func MakeWallet(ctx context.Context, config *WalletConfig) (Wallet, error) {
 		}
 		pChainTxs[txID] = tx
 	}
-
+	subnetOwners, err := ExtractTxSubnetOwners(ctx, avaxState.PClient, pChainTxs)
```

Comment:
Should we replace `config.PChainTxs` and `config.PChainTxsToFetch` with `config.SubnetIDs`? The reason that we were passing in `config.PChainTxsToFetch` previously was to avoid performing any network calls if we already had the owner... But we aren't doing that anymore (as we are just converting them to their `subnetID`s... It feels like this flow could be simplified.

Then `ExtractTxSubnetOwners` could become `GetSubnetOwners(ctx, avaxState.PClient, subnetIDs...)`

Reply:
I agree the simplification is fine. I was assuming there was a future need for other tx types in the wallet, and didn't want to change primary wallet API. Implementing the suggestion.